fairseq/data/masked_lm_dataset.py

```diff
@@ -228,7 +228,10 @@ def _collate(
 
                 # mask according to specified probabilities.
                 masked_blk_one, masked_tgt_one = self._mask_block(
```

Comment:
`_mask_block` also gets called below for the block 2. 
maybe we can just use the member variable directly in `_mask_block()` function and not accept them as args ?